src/constants/common.js

```diff
@@ -37,4 +39,15 @@ module.exports = {
 	RESOURCE_CREATOR: 'resource_creator',
 	ADMIN_ROLE: 'admin',
 	ORG_ADMIN_ROLE: 'org_admin',
+	CERTIFICATE_PATH: 'certificate/',
+	LOGO_PATH: 'certificate_logo/',
+	SIGNATURE_PATH: 'certfile_signature/',
+	BASETEMPLATE_PATH: 'certfile_basetemplate/',
+	PROJECT_PATH: 'project/',
+	BASETEMPLATE: 'baseTemplate',
+	SIGNATURE: 'signature',
+	LOGO: 'logo',
+	CERTIFICATE: 'certificate',
+	NO_OF_MINUTES: 30,
```

Comment:
@ankit1shahu change the variable name its confusing if its related to link expiry add a relevant name. same for NO_OF_EXPIRY_TIME we need to understand what expiry time it is